system-test/.eslintrc.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Since the sources are all in TypeScript, I don't think we need this file.  I could be wrong though!